lib/ziti-tunnel-cbs/include/ziti/ziti_dns.h

```diff
@@ -31,7 +31,8 @@ typedef int (*dns_fallback_cb)(const char *name, void *ctx, struct in_addr* addr
 
 int ziti_dns_setup(tunneler_context tnlr, const char *dns_addr, const char *dns_cidr);
 
-void ziti_dns_set_fallback(struct uv_loop_s *l, dns_fallback_cb fb, void *ctx);
```

Comment:
The `dns_fallback_cb` typedef (on line 30) is no longer referenced - should it be removed?